### PR TITLE
(DOCSP-19165): Intro for Backend Devs Readability Refactor + Small Web Intro Tweaks

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

Edit the backend developer introduction page for readability. 

To assess readability used the tools:  
- https://hemingwayapp.com/
- http://readable.com/

Refactored so that it achieved score on Readable of 'B', per [direction in the ticket's epic](https://jira.mongodb.org/browse/DOCSP-19160?focusedCommentId=4175080&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4175080).

Also included some slight content tweaks to the web dev intro page b/c i forgot to push some small changes before merging https://github.com/mongodb/docs-realm/pull/1474 

### Jira

- https://jira.mongodb.org/browse/DOCSP-19165

### Staged Changes (Requires MongoDB Corp SSO)

- [Introduction to MongoDB Realm for Backend Developers](https://docs-mongodborg-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-19165/get-started/introduction-backend)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/get-started/introduction-backend.txt [hunk 1]
```diff
@@ -10,11 +10,11 @@ Introduction to MongoDB Realm for Backend Developers
...
 
-As a backend developer, you can use {+service+} to rapidly develop cloud-based
-applications that can react to changes in your `{+atlas+}
-<https://www.mongodb.com/realm?tck=docs_realm#atlas-form-container>`_ data, connect that data
-to other systems and services, and scale to meet demand without the need for
-managing your own database and server infrastructure.
+As a backend developer, you can use {+service+} to rapidly develop cloud-based applications.
+Realm apps can react to changes in your `{+atlas+}
+<https://www.mongodb.com/realm?tck=docs_realm#atlas-form-container>`_ data, connect that data to other systems,
+and scale to meet demand. {+service-short+} does all this without the need
```
Comment (cbush) [inline source/get-started/introduction-backend.txt:16]:
```suggestion
and scale to meet demand. {+service-short+} does this without the need 
```

File: source/get-started/introduction-backend.txt [hunk 2]
```diff
@@ -27,64 +27,60 @@ What Problem Does MongoDB Realm Solve?
 --------------------------------------
 
 It takes time, resources, and expertise to build, administer, and maintain
-backend infrastructure. User authentication and access control are common but
-complex systems that you do not need to write from scratch. :term:`{+service+}`
-manages all parts of the backend infrastructure and user management for you,
-enabling you to focus on what makes your app special so you can get it to your
-users faster.
+backend infrastructure. However, a lot of these features are very similar across applications.
```
Comment (cbush) [inline source/get-started/introduction-backend.txt:30]:
```suggestion
backend infrastructure. However, a lot of these features are similar across applications.
```

File: source/get-started/introduction-backend.txt [hunk 3]
```diff
@@ -27,64 +27,60 @@ What Problem Does MongoDB Realm Solve?
...
   - Run cloud-based :term:`{+service-short+} Functions <{+service-short+}
-    Function>` that can crunch data or interact with third-party services, such
-    as Slack, Twilio, or Twitter. You can integrate virtually any Node.js module
-    in your functions with :ref:`dependency resolution <external-dependencies>`.
-  - Skip writing an additional API on top of your {+service+} service --
-    authenticated clients may directly interact with the parts of the database
-    they have permission to access.
+    Function>` that can crunch data or interact with external services like
+    Slack, Twilio, and Twitter. You can integrate Node.js modules
+    in your functions with dependency resolution.
```
Comment (cbush) [inline source/get-started/introduction-backend.txt:42]: Since third-party services are going away, maybe we should just say "that can integrate with Node.js modules."

File: source/get-started/introduction-backend.txt [hunk 4]
```diff
@@ -27,64 +27,60 @@ What Problem Does MongoDB Realm Solve?
...
+  - Skip writing an authentication API on top of your {+service+} service.
+    Authenticated clients can interact with the parts of the database based on their permissions.
```
Comment (cbush) [inline source/get-started/introduction-backend.txt:44]: "With the parts" makes no sense anymore. Maybe:
```suggestion
    Authenticated clients can interact with parts of the database based on their permissions.
```

File: source/get-started/introduction-backend.txt [hunk 5]
```diff
@@ -27,64 +27,60 @@ What Problem Does MongoDB Realm Solve?
...
-  - Pay for and receive only the exact amount of compute you need at any given
-    time with {+service+}'s usage-based pricing model. Usage under a certain
-    amount per month is always free.
+  - Get up and running quickly for free, then scale according to the demands of your application.
+  - Pay for and receive only the exact amount of compute you need at any given time
+    with usage-based pricing. Usage under a certain amount per month is always free.
 
 Simply secure:
-  - Allow users to log in with familiar :term:`authentication providers
+  - Let log in with familiar :term:`authentication providers
```
Comment (cbush) [inline source/get-started/introduction-backend.txt:55]: Forgot a word?
```suggestion
  - Let users log in with familiar :term:`authentication providers
```

File: source/get-started/introduction-backend.txt [hunk 6]
```diff
@@ -27,64 +27,60 @@ What Problem Does MongoDB Realm Solve?
...
+    Define the :ref:`rules and roles <data-access-permissions>` declaratively or with a function.
   - Enforce data integrity by defining :doc:`validation logic
     </mongodb/enforce-a-document-schema>` with :term:`{+backend-schema+}`.
 
-You can make entirely cloud-based applications with {+service+} or tie your
-{+app+} into mobile or web clients and seamlessly sync data across devices with
-:term:`{+sync+}`.
+Synchronized:
+  - Seamlessly sync data across devices with :term:`{+sync+} <{+sync+}>`.
+  - Avoid writing complex synchronization logic with the :ref:`Database SDKs <realm-sdks>`.
```
Comment (cbush) [inline source/get-started/introduction-backend.txt:64]: Is this section relevant to "backend developers"?

**Review by cbush — APPROVED** [review]
LGTM with a few comments.

File: source/get-started/introduction-backend.txt (hunk 6 above)
Comment (mongodben) [reply source/get-started/introduction-backend.txt:64]: it'd be the frontend/mobile devs using, but figured worth mentioning here as ti saves backend devs writing lots of synchronization code (b/c SDKs does for them).
